pypif/pif.py

```diff
@@ -26,51 +26,53 @@ def dumps(pif, **kwargs):
     return json.dumps(pif, cls=PifEncoder, **kwargs)
 
 
-def load(fp, **kwargs):
+def load(fp, builder=None, **kwargs):
```

Comment:
Is builder a common name for this?

Reply:
I don't think so.  `class`, `type`, and `object` are python keywords.  Open to suggestions.

Reply:
I see clazz used in java world very commonly.

Reply:
https://stackoverflow.com/questions/38718484/whats-convention-for-naming-a-class-or-method-as-class-in-python